services/ux-backend/main.go

```diff
@@ -94,15 +96,18 @@ func main() {
 	// TODO: remove '/onboarding-tokens' in the future
 	http.HandleFunc("/onboarding-tokens", func(w http.ResponseWriter, r *http.Request) {
 		// Set the Deprecation header
+		ctx := context.Background()
 		w.Header().Set("Deprecation", "true") // Standard "Deprecation" header
 		w.Header().Set("Link", "/onboarding/client-tokens; rel=\"alternate\"")
-		clienttokens.HandleMessage(w, r, config.tokenLifetimeInHours, cl, namespace)
+		clienttokens.HandleMessage(ctx, w, r, config.tokenLifetimeInHours, cl, namespace)
 	})
 	http.HandleFunc("/onboarding/client-tokens", func(w http.ResponseWriter, r *http.Request) {
-		clienttokens.HandleMessage(w, r, config.tokenLifetimeInHours, cl, namespace)
+		ctx := context.Background()
+		clienttokens.HandleMessage(ctx, w, r, config.tokenLifetimeInHours, cl, namespace)
 	})
 	http.HandleFunc("/onboarding/peer-tokens", func(w http.ResponseWriter, r *http.Request) {
-		peertokens.HandleMessage(w, r, config.tokenLifetimeInHours, cl, namespace)
+		ctx := context.Background()
+		peertokens.HandleMessage(ctx, w, r, config.tokenLifetimeInHours, cl, namespace)
```

Comment:
as already mentioned in one of earlier comments, http.Request already provides a context and use that inside the handler but not create the ctx everytime as being done here.

Reply:
@leelavg @mrudraia1 
Let me clarify. Evey request should handle its own context. If the req itself provide a context we should use that, if not we should create a new context at the topmost level of the request and pass it down the call chain

Reply:
> Every request should handle its own context.
- yes, that's what I mentioned earlier in another comment.

Reply:
Do, I need to change this to original like

peertokens.HandleMessage( r.context,  .......)

Reply:
ok